hknweb/events/templates/events/event_update.html

```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+{% load static %}
+{% block title %}Edit {{ event.name }}{% endblock %}
+
+{% block header %}
+<link rel="stylesheet" href="{% static "events/style.css" %}">
+<style>
+    .heading {
+        display: none;
+    }
+    .content {
+        padding-top: 1em;
+        padding-bottom: 1em;
+    }
+    @media screen and (min-width: 1000px) {
+        .content {
+            padding-top: 1.5em;
+            padding-bottom: 1.5em;
+        }
+    }
+    .centered {
+        max-width: 700px;
+        margin: 0 auto;
+    }
+</style>
+{% endblock %}
+
+{% block heading %}Edit Event: {{ event.name }}{% endblock %}
+
+{% block content %}
+<h1 id="add-event-title"> Edit Event: {{ event.name }} </h1>
+
+<div class="centered">
+    <form method="post">
+        {% csrf_token %}
+        {{ form.as_p }}
+        <input type="submit" value="Update">
+        {% if messages %}
+            {% for message in messages %}
+                <br>
+                <span class="message">{{ message }}</span>
+            {% endfor %}
+        {% endif %}
```

Comment:
Should we be putting these messages into the `base.html`? These seem pretty common.